ONSdigital/dp-frontend-dataset-controller
# Feature/census landing page skeleton

### What

- Started implementing skeleton of census dataset landing page based on prototype as it currently stands - all still a WIP
- This skeleton includes programmatic templates (i.e. using dummy data mapped to a page model)
- There are some components that have not yet been finalised, and thus are not in the current templates. These are: table of contents, 'get data', variables table, datestamp, share dataset
- All is feature flagged
- There are some styling tweaks still required, such as with the collapsible "Why is this useful" section - but these are waiting on changes being made to the design system (namespacing, primarily)

### How to review

- Run this app with the feature flag, `ENABLE_CENSUS_PAGES=true`
- Check the changes locally by visiting `/census/datasets` (route subject to change)

### Who can review

Anyone
